### DISCUSSION TITLE
Can't search

### DISCUSSION THREAD

**Discussion by Ekki2015** [z-huang/music#179] [General]
It''s not possible to search, it stopped working today. 😕 I have tried several times to clear the memory as well

https://user-images.githubusercontent.com/12226562/190510906-5e44e7e7-8b84-44a8-a42b-40f1fa269802.mp4

**Comment by z-huang** [comment]
It seems that YouTube changed the API response, so the parsing is broken.

**Comment by z-huang** [comment]
Fixed in 29cfb3a4559554f7696ae0b81e50d80eec453c7f.
